packages/shared/tailwind.config.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
           tertiary: 'var(--theme-background-tertiary)',
           notification: 'var(--theme-background-notification)',
           'cabbage-blur': 'var(--theme-background-cabbage-blur)',
+          'secondary-blur': 'var(--theme-background-secondary-blur)',
```

Comment:
I have to check with design for correct blur values, current ones seem to dark with settings. Just needs some adjustments. This might not be needed at the end.